pkg/k8shandler/secret.go

```diff
@@ -92,3 +94,33 @@ func (clusterRequest *ClusterLoggingRequest) RemoveSecret(secretName string) err
 
 	return nil
 }
+
+func calcSecretHashValue(secret *core.Secret) (string, error) {
```

Comment:
this is something that we can confirm with a test

Reply:
what would we be confirming?

Reply:
the value produced is the value expected.  What if someone comes along and removes the sort statement.  It's likely immaterial given you would drive the change to the new value of the algo of the function but asking the question